### PR TITLE
fix/pg pool connection timeout

### PR DESCRIPTION
**Pull request by sywhb**
- **get postgres connection pool size from env**
- **connection idle timeout in 10 seconds**

### CODE REVIEW

File: packages/api/src/queue-processor.ts [hunk 1]
```diff
@@ -153,8 +153,12 @@ const main = async () => {
     subscribers: [__dirname + '/events/**/*{.js,.ts}'],
     namingStrategy: new SnakeNamingStrategy(),
     logger: new CustomTypeOrmLogger(['query', 'info']),
-    connectTimeoutMS: 40000, // 40 seconds
+    connectTimeoutMS: 10000, // 10 seconds
     maxQueryExecutionTime: 10000, // 10 seconds
+    extra: {
```
Comment (jacksonh) [inline packages/api/src/queue-processor.ts:158]: are we losing this `options: process.env.PG_EXTRA_OPTIONS,` ? Right now we use that to set the lock timeout.  its set like this in k8s `PG_EXTRA_OPTIONS: '-c lock_timeout=500ms'`

Comment (sywhb) [reply packages/api/src/queue-processor.ts:158]: > are we losing this `options: process.env.PG_EXTRA_OPTIONS,` ? Right now we use that to set the lock timeout. its set like this in k8s `PG_EXTRA_OPTIONS: '-c lock_timeout=500ms'`

no, we won't but i noticed we didn't have it for queue-processor in the first place. did you do it on purpose?

Comment (sywhb) [reply packages/api/src/queue-processor.ts:158]: the option was set in api-backend only

Comment (jacksonh) [reply packages/api/src/queue-processor.ts:158]: Oh, yeah I think we will want it everywhere

Comment (jacksonh) [reply packages/api/src/queue-processor.ts:158]: Currently i think the AppEngine queue-processor is using it

Comment (sywhb) [reply packages/api/src/queue-processor.ts:158]: but it is not in the code

Comment (sywhb) [reply packages/api/src/queue-processor.ts:158]: ohh, you set it on cloud sql

Comment (sywhb) [reply packages/api/src/queue-processor.ts]: @jacksonh i have removed the `setOptions` in queue-processor

**Review by jacksonh — APPROVED** [review]

**Pull request merged**